exercises/isogram/example.h

```diff
@@ -0,0 +1,7 @@
+#include <string>
```

Comment:
This needs include guards like the other exercises

Reply:
Done. Good point!